index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Weather Report</title>
+    <link rel="stylesheet" href="styles/index.css">
+</head>
+<body>
+    
+    <main>
+        <button id="dark-mode">Dark mode 🌌</button>
+        <header class="header_header">
+            <h1>Weather Report</h1>
+            
+            <span>
+                For the lovely city of 
+                <span id="headerCityName" class="header_city-name">
+                    San Francisco
+                </span>
+            </span>
+        </header>
+
+        <section class="city__section">
+            <h2>City Name</h2>
+            <input type="text" id="cityNameInput" value="San Francisco">
+            <button id="cityNameReset" class="city-name-reset-btn">Reset</button>
+        </section>
+
+        <section class="temperature__section">
+            <h2>Temperature</h2>
+            <div id = "container">            
+                <h1 id="temp" class="orange">71</h1>
```

Comment:
Your initial class here is `orange` but 71 is a `green` color.

```suggestion
                <h1 id="temp" class="green">71</h1>
```